sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcHadoopFsRelationSuite.scala

```diff
@@ -82,44 +80,4 @@ class OrcHadoopFsRelationSuite extends HadoopFsRelationTest {
       }
     }
   }
-
-  test("SPARK-13543: Support for specifying compression codec for ORC via option()") {
-    withTempPath { dir =>
-      val path = s"${dir.getCanonicalPath}/table1"
-      val df = (1 to 5).map(i => (i, (i % 2).toString)).toDF("a", "b")
-      df.write
-        .option("compression", "ZlIb")
-        .orc(path)
-
-      // Check if this is compressed as ZLIB.
-      val maybeOrcFile = new File(path).listFiles().find { f =>
-        !f.getName.startsWith("_") && f.getName.endsWith(".zlib.orc")
-      }
-      assert(maybeOrcFile.isDefined)
-      val orcFilePath = maybeOrcFile.get.toPath.toString
-      val expectedCompressionKind =
-        OrcFileOperator.getFileReader(orcFilePath).get.getCompression
-      assert("ZLIB" === expectedCompressionKind.name())
-
-      val copyDf = spark
-        .read
-        .orc(path)
-      checkAnswer(df, copyDf)
-    }
-  }
-
-  test("Default compression codec is snappy for ORC compression") {
-    withTempPath { file =>
-      spark.range(0, 10).write
-        .orc(file.getCanonicalPath)
-      val expectedCompressionKind =
-        OrcFileOperator.getFileReader(file.getCanonicalPath).get.getCompression
```

Comment:
`OrcFileOperator` is defined in `sql\hive`.

Reply:
@gatorsmile . This test case should be tested on `native` implementation, too.
`HiveOrcHadoopFsRelationSuite` test coverage is only `hive` implementation.